example/App.js

```diff
@@ -40,7 +40,7 @@ export default () => {
         expiryYear: payload.expiryYear || '??',
         issuer: issuer,
         cvc: payload.cvc || '??',
-        cardholderName: payload.cardholderName || '??',
+        cardholderName: payload.name || '??',
```

Comment:
I think this will break android. Android currently sets `cardholderName`

Reply:
okay cool ill change it back :-)